samples/cpp/benchmark_app/main.cpp

```diff
@@ -36,20 +36,24 @@
 #include "statistics_report.hpp"
 #include "utils.hpp"
 
-#if defined _WIN32
+#if defined(_WIN32)
 #include <windows.h>
 #include <psapi.h>
-#else
+#elif  defined(__APPLE__) && defined(__MACH__)
```

Comment:
This `&&` suggests there is a configuration with \_\_APPLE__ but without \_\_MACH__ which you don't want to support. Is that so? Why isn't it supported?

I'd expect this to be just `#elif defined(__APPLE__)`

Reply:
I don't mind leaving only `__APPLE__`, please let me know how to do it properly from OV's point of view.

Reply:
OpenVINO doesn't seem to be consistent on that. Leave only `__APPLE__` to avoid questions similar to 

> This `&&` suggests there is a configuration with __APPLE__ but without __MACH__ which you don't want to support. Is that so? Why isn't it supported?

Reply:
Done